app/src/main/java/com/example/miguelflores/espressotest/activities/ShowBundleActivity.java

```diff
@@ -0,0 +1,37 @@
+package com.example.miguelflores.espressotest.activities;
+
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v7.app.AppCompatActivity;
+import android.view.MenuItem;
+import android.widget.TextView;
+
+import com.example.miguelflores.espressotest.R;
+
+public class ShowBundleActivity extends AppCompatActivity {
+
+    Bundle bundle;
```

Comment:
you dont need to keep the bundle here neither the text view for this example1 ;)